engine/src/main/resources/assets/i18n/menu_fa.lang

```diff
@@ -0,0 +1,244 @@
+{
+    "activate-module": "فعال کردن",
+    "add-server": "جدید اضافه کردن",
+    "add-server-tip": "شما می توانید با فشار دکمه ی اینتر اسم را در ادرس مورد نظر کپی کنید.",
+    "advanced-usage": "کاربرد پیشرفته",
+    "animate-grass": "خلق علفی کاملا طبیعی و زنده",
+    "animate-water": "خلق اب طبیعی",
+    "audio-settings": "تنظیمات صدا",
+    "audio-settings-title": "عنوان تنظیمات صدا",
+    "back": "بازگشت",
+    "binding-attack": "حمله",
+    "binding-autoMove-mode": "کلید خود جرکت",
+    "binding-backwards": "به عقب",
+    "binding-behavior-editor": "دستور تنظیم ویرایشگر",
+    "binding-chat": "کلید چت",
+    "binding-console": "کلید فرمان",
+    "binding-crouch": "قوز کردن / پایین رفتن",
+    "binding-crouch-mode": "حالت نشسته",
+    "binding-decrease-view-distance": "کاهش فاصله ی دید",
+    "binding-forwards": "رو به جلو",
+    "binding-frob": "هدف استفاده",
+    "binding-hide-hud": "اچ یو دی مخفی",
+    "binding-increase-view-distance": "افزایش فاصله ی دید",
+    "binding-jump": "پرش",
+    "binding-left": "گردش از چپ",
+    "binding-nui-editor": "NUI Editor",
+    "binding-nui-skin-editor": "ان. یو. آی ادیتور اسکین",
+    "binding-pause": "منوی انتخاب در حین بازی",
+    "binding-right": "گردش از راست",
+    "binding-show-online-players": "نمایش بازیکنان انلاین",
+    "binding-toggle-speed-permanently": "دکمه ی سرعت دایمی",
+    "binding-toggle-speed-temporarily": "دکمه ی سرعت موقتی",
+    "binding-use-item": "استفاده از آیتم نگه داشته شده",
+    "bloom-effect": "بلووم                        ⁠⁠",
+    "bobbing": "حرکت به بالا و پایین",
+    "camera-blur": "تیرگی دوربین",
+    "camera-blur-max": "تیرگی دوربین در بالاترین حد",
+    "camera-blur-normal": "تیرگی دوربین در حد طبیعی",
+    "camera-blur-off": "خاموش کردن تیرگی دوربین",
+    "camera-blur-some": "Some",
+    "camera-details-settings": "تنظیمات جزییات گرافیکی دوربین",
+    "camera-film-grain": "بافت فیلم",
+    "camera-setting": "تنظیمات دوربین",
+    "camera-setting-cinematic": "سینمایی",
+    "camera-setting-normal": "تنظیمات معمولی دوربین",
+    "camera-setting-smooth": "نرم",
+    "category-behavior": "دسته بندی عمومی رفتار",
+    "category-general": "دسته بندی عمومی رفتار",
+    "category-interaction": "اثر متقابل",
+    "category-inventory": "دسته بندی -بازبینی",
+    "category-mouse": "موش واره",
+    "category-movement": "حرکت",
+    "category-nui": "NUI",
+    "change-keybind-popup-message": "is already bound to an action. Rebind anyway?",
```

Comment:
This is untranslated - can you wrap this up just so the language is 100% finished? [2]